.github/workflows/build.yml

```diff
@@ -15,7 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
+    - name: install dependencies
+      run: |
+        sudo apt update
+        sudo apt-get install -y --no-install-recommends libgbm-dev
```

Comment:
Could you add a comment why we need this?

Reply:
I'm adding the GH issue where the fix comes from.